signer/src/main.rs

```diff
@@ -121,9 +122,12 @@ async fn run_stacks_event_observer(ctx: &impl Context) -> Result<(), Error> {
         .route("/new_block", post(api::new_block_handler))
         .with_state(state);
 
+    let config = ctx.config().signer.event_observer.clone();
+
     // run our app with hyper
     // TODO: This should be read from configuration
```

Comment:
I guess this TODO should be removed now.

Reply:
Haha whooops, done!